epregressions/runtests.py

```diff
@@ -179,6 +187,67 @@ def read_file_content(file_path):
             idf_text = f_idf.read()
         return idf_text
 
+    @staticmethod
+    def add_or_modify_output_sqlite(idf_text, force_output_sql: ForceOutputSQL,
+                                    force_output_sql_unitconv: ForceOutputSQLUnitConversion,
+                                    isEpJSON: bool = False):
+        """Will add or modify the Output:SQLite object based on the provided enums that corresponds to the 'Option'"""
+        # Ensure we deal with the enum
+        if not isinstance(force_output_sql, ForceOutputSQL):
+            raise ValueError("Expected an Enum ForceOutputSQL, not {}".format(force_output_sql))
+
+        if not isinstance(force_output_sql_unitconv, ForceOutputSQLUnitConversion):
+            raise ValueError("Expected an Enum ForceOutputSQLUnitConversion, not "
+                             "{}".format(force_output_sql_unitconv))
+
+        if isEpJSON:
+            data = json.loads(idf_text)
+            if "Output:SQLite" in data and len(data["Output:SQLite"]) >= 1:
+                sqlite_obj = data["Output:SQLite"][list(data["Output:SQLite"].keys())[0]]
+
+            else:
+                data["Output:SQLite"] = {"Output:SQLite 1": {}}
+                sqlite_obj = data["Output:SQLite"]["Output:SQLite 1"]
+
+            sqlite_obj['option_type'] = force_output_sql.value
+            if force_output_sql_unitconv != ForceOutputSQLUnitConversion.NOFORCE:
+                sqlite_obj['unit_conversion'] = force_output_sql_unitconv.value
+
+            return json.dumps(data, indent=4)
```

Comment:
New EPSJON handling. It's a json, so read it, do shenanigans, then dump it back to a string.

Reply:
:+1: